src/transformers/models/led/modeling_led.py

```diff
@@ -174,8 +174,7 @@ def forward(
 
         The `attention_mask` is changed in :meth:`BertModel.forward` from 0, 1, 2 to -ve: no attention
 
-              0: local attention
-            +ve: global attention
+              0: local attention +ve: global attention
```

Comment:
Not sure how to format those as I don't understand this docstring.

Reply:
add `- ` bullet points?
```
-  0: local attention
- +ve: global attention

Reply:
I can do that, but +ve does not make any sense to me, so I don't know what was the original author's idea behind that docstring. Asked patrick since he wrote that code.